webapp/portlet/src/main/webapp/vue-apps/common/components/ExoActivityRichEditor.vue

```diff
@@ -143,6 +152,12 @@ export default {
     }
   },
   mounted() {
+    if (this.value === '') {
+      const storageMessage =  localStorage.getItem(`activity-message-${this.contextName}`);
+      const storageMessageObject =  storageMessage && JSON.parse(storageMessage) || {};
+      const storageMessageText = storageMessageObject?.url === eXo.env.server.portalBaseURL && storageMessageObject?.text || '';
+      this.value = storageMessageText;
```

Comment:
```suggestion
      this.value = storageMessageObject?.url === eXo.env.server.portalBaseURL && storageMessageObject?.text || '';
```

Besides, `this.value` is a `props` attribute, shouldn't we add a second parameter to `initCKEditor` for default value to use ?